src/scenes/Partners/Detail/Tabs/Profile/PartnerContactCard.tsx

```diff
@@ -53,16 +55,34 @@ export const AddressCard = ({
   const canEdit = canEditAny(partner, false, 'address');
 
   return (
-    <Card className={className}>
-      <CardActionArea
-        onClick={onEdit}
-        className={classes.cardContent}
-        disabled={!canEdit}
-      >
+    <section>
+      <CardActions className={classes.cardActions}>
+        <Typography variant="h3">
+          {partner ? 'Contact Information' : <Skeleton width="120px" />}
+        </Typography>
+        {partner && (
+          <Tooltip title="Edit Address">
+            <IconButton
+              aria-label="Edit Address"
+              disabled={!canEdit}
+              onClick={onEdit}
+            >
+              <Edit />
+            </IconButton>
+          </Tooltip>
+        )}
+      </CardActions>
+      <Card className={className}>
         <CardContent className={classes.cardContent}>
```

Comment:
wouldn't an EditableSection component need to take a different data shape as input (partner in this case but it could be any shape for other use cases)